controllers/ProjectController.php

```diff
@@ -172,11 +168,17 @@ public function actionCreate() {
             
         } else { //Sinon c'est qu'il faut afficher ce formulaire
             //Récupération de la liste des projets existants pour la dropdownList
+            //SILEX:conception
+            // This quick fix is used to show all users available in the 
+            // dropdown list but we need to implements an autocompletion
+            // search linked to the web service instead of load all
+            // users in a list
+            //SILEX:conception
             $searchModel = new ProjectSearch();
-            $projects = $searchModel->find($sessionToken,[]);
+            $projects = $searchModel->find($sessionToken, ["pageSize" => 100]);
```

Comment:
is this quick fix  really necessary ? 
It's better to fix with the web service.

Reply:
need to implements an autocompletion search linked to the web service instead of load all